### PR TITLE
Remove clovis support code

### PR DESCRIPTION
**Pull request by deepanjanroy**
Remove things added to lighthouse code to make clovis work that are no longer necessary.

### CODE REVIEW

File: src/lib/network-recorder.js [hunk 1]
```diff
@@ -54,42 +52,37 @@ class NetworkRecorder {
...
     // data.errorText -> localizedDescription
     this.networkManager._dispatcher.loadingFailed(data.requestId,
         data.timestamp, data.type, data.errorText, data.canceled,
         data.blockedReason);
   }
 
-  onResourceChangedPriority(data) {
-    this._rawEvents.push({method: 'Network.resourceChangedPriority', params: data});
+  onResourceChangedPriority() {
+    // TODO: Add call to dispatcher after devtools dependency is updated
```
Comment (brendankenny) [inline src/lib/network-recorder.js]: not sure how to interpret what this TODO is asking for, but make sure to file an issue for this :)

**Comment by brendankenny** [comment]
LGTM!

File: src/lib/network-recorder.js (hunk 1 above)
Comment (deepanjanroy) [inline src/lib/network-recorder.js]: Filed https://github.com/GoogleChrome/lighthouse/issues/329

**Comment by paulirish** [comment]
Going to rebase this and land it...

**Comment by paulirish** [comment]
replaced by #355

**Pull request closed without merging**